dune-project

```diff
@@ -0,0 +1,113 @@
+(lang dune 2.4)
+
+; Since we want to generate opam files we need to provide informations ;
+(generate_opam_files true)
+
+(name alt-ergo)
+(version dev)
+(license "OCamlPro Non-Commercial Purpose License, version 1, LGPL-2.1-only")
+(authors "Alt-Ergo developers")
+(maintainers "Alt-Ergo developers")
+(source (github OCamlPro/alt-ergo))
+; Not specifying bug_reports since it defaults to the github issue page
+(homepage "http://alt-ergo.ocamlpro.com/")
+(documentation "https://ocamlpro.github.io/alt-ergo/dev/")
+
+; Formatting is automatically enabled if using dune 2.0 or greater
+; Usage:
+; dune build @fmt
+; If the result is ok:
+; dune
+; Or simply (available through make fmt):
+; dune build @fmt --auto-promote
+
+(using menhir 2.0)
+
+; First package, the alt-ergo binary
+
+(package
+ (name alt-ergo)
+ (synopsis "The Alt-Ergo SMT prover")
+ (description "Alt-Ergo is an automatic theorem prover of mathematical formulas. It was developed at LRI, and is now maintained at OCamlPro.
+
+See more details on http://alt-ergo.ocamlpro.com/")
+ (depends
+  (ocaml (>= 4.04.0))
+  (dune (>= 2.4))
+  (alt-ergo-lib (= version))
+  (alt-ergo-parsers (= version))
+ )
+)
+
+; Second package, the alt-ergo gui binary
+
+(package
+ (name altgr-ergo)
+ (synopsis "The GUI for the Alt-Ergo SMT prover")
+ (description "Altgr-Ergo is the graphical interface for the Alt-Ergo SMT prover.
+
+Alt-Ergo is an automatic theorem prover of mathematical formulas. It was developed at LRI, and is now maintained at OCamlPro.
+
+See more details on http://alt-ergo.ocamlpro.com/"
+ )
+ (license "OCamlPro Non-Commercial Purpose License, version 1, LGPL-2.1-only")
+
+ (depends
+  (ocaml (>= 4.04.0))
+  (dune (>= 2.4))
+  (alt-ergo (= :version))
+  (alt-ergo-lib (= :version))
+  (alt-ergo-parsers (= :version))
+  lablgtk
+  conf-gtksourceview
+ )
+)
+
+; Third package, the alt-ergo parsers library
+
+(package
+ (name alt-ergo-parsers)
+ (synopsis "The Alt-Ergo SMT prover parser library")
+ (description "This is the parser library used in the Alt-Ergo SMT solver.
+
+Alt-Ergo is an automatic theorem prover of mathematical formulas. It was developed at LRI, and is now maintained at OCamlPro.
+
+See more details on http://alt-ergo.ocamlpro.com/"
+ )
+ (license "OCamlPro Non-Commercial Purpose License, version 1, LGPL-2.1-only")
```

Comment:
Only the Package Alt-Ergo have both OCP and LGPL licences due to the AB-Why3 plugin.